static/src/stylesheets/layout/nav/_cta-bar.scss

```diff
@@ -32,7 +16,7 @@
     }
 
     @include mq(desktop) {
-        max-width: 310px;
+        max-width: 320px;
```

Comment:
at smallest desktop currently the copy runs onto 2 lines. This change makes it consistent with DCR